### PR TITLE
Improve documentation for local docs build

### PR DESCRIPTION
**Pull request by stes**
This PR makes it easier to locally build and test the documentation.

The components are:
- Update the contribution guide which will get published on https://cebra.ai/docs/contributing.html
- Add a script at `./tools/build_docs.sh` which will check-out all additional repositories, trigger a docs build, and start a webserver to inspect the build at http://127.0.0.1:8080/

### CODE REVIEW

**Review by MMathisLab — APPROVED** [review]
Lgtm 🦓🎉

**Review by MMathisLab — APPROVED** [review]

**Comment by MMathisLab** [comment]
(Just a simple permission issue it seems)

File: Makefile [hunk 1]
```diff
@@ -57,7 +57,7 @@ interrogate:
 
 # Build documentation using sphinx
 docs:
-	(cd docs && PYTHONPATH=.. make page)
+	./tools/build_docs.sh
```
Comment (stes) [inline Makefile:60]:
```suggestion
	(cd docs && PYTHONPATH=.. make page)
```

**Comment by stes** [comment]
@clabot

**Comment by stes** [comment]
@clabot check

**Pull request merged**